parsec/interfaces/ptg/ptg-compiler/jdf2c.c

```diff
@@ -4708,12 +4727,17 @@ static void jdf_generate_constructor( const jdf_t* jdf )
     pi.idx = 0;
     JDF_COUNT_LIST_ENTRIES(jdf->functions, jdf_function_entry_t, next, pi.maxidx);
     {
-        char* prof = UTIL_DUMP_LIST(sa1, jdf->functions, next,
-                                    dump_profiling_init, &pi, "", "    ", "\n", "\n");
+        int need_profile = 0;
```

Comment:
If I understand this code correctly it is meant to save us 2 integers per task class, but it makes the code more complicated to read. Not sure this is worth the effort.